platform/vs/docker-gbsyncd-vs/critical_processes

```diff
@@ -1 +0,0 @@
-program:syncd
```

Comment:
Rather than removing this, maybe syncd could exit with an "expected" exit code, or simply disable the feature altogether on platforms without a gearbox.

Reply:
i cannot find example that setup feature based on the platform. also here the platform is not asic platform, it is the onie platform here. need more dynamic version. I do not think the current infrastructure supports it.

Reply:
@lguohan: So do you suggest removing this and revisiting later?

Reply:
i will add it back. check pr. https://github.com/Azure/sonic-sairedis/pull/676